src/elements/content-preview/ContentPreview.js

```diff
@@ -1312,64 +1313,77 @@ class ContentPreview extends React.PureComponent<Props, State> {
         /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
         return (
             <Internationalize language={language} messages={messages}>
-                <div id={this.id} className={styleClassName} ref={measureRef} onKeyDown={this.onKeyDown} tabIndex={0}>
-                    {hasHeader && (
-                        <PreviewHeader
-                            file={file}
-                            logoUrl={logoUrl}
-                            token={token}
-                            onClose={onHeaderClose}
-                            onPrint={this.print}
-                            canDownload={this.canDownload()}
-                            canPrint={canPrint}
-                            onDownload={this.download}
-                            contentAnswersProps={contentAnswersProps}
-                            contentOpenWithProps={contentOpenWithProps}
-                            canAnnotate={this.canAnnotate()}
-                            selectedVersion={selectedVersion}
-                        />
-                    )}
-                    <div className="bcpr-body">
-                        <div className="bcpr-container" onMouseMove={this.onMouseMove} ref={this.containerRef}>
+                <APIContext.Provider value={(this.api: any)}>
```

Comment:
Shold we be more explicit with the value type. this.api should be of type APIFactory right?